src/viz/test/CMakeLists.txt

```diff
@@ -21,12 +21,13 @@ file( GLOB test_sources
 # Build Unit tests
 # ---------------------------------------------------------------------------- #
 
-if( NOT APPLE )
+if( NOT APPLE AND DRACO_C4 STREQUAL MPI )
```

Comment:
Nice!  Since the CI tests the `DRACO_C4=SCALAR`, I'll assume this change is safe.

Reply:
Yap, it doesn't expressly test these in scalar mode anymore (because I made them parallel tests). The scalar versions of the test messed with python diff scripts. If we really need the scalar tests I can try and sort them out.